rcl/src/rcl/node.c

```diff
@@ -39,6 +42,10 @@ extern "C"
   _snprintf_s(buffer, buffer_size, _TRUNCATE, format, __VA_ARGS__)
 #endif
 
+static const char * sros_root_var_name = "ROS_SECURE_ROOT";
+static const char * sros_security_strategy_var_name = "ROS_SECURITY_STRATEGY";
+static const char * sros_enable_security_var_name = "ROS_ENABLE_SECURITY";
```

Comment:
I think it would be better if all SROS specific environment variable share a common prefix, e.g. `ROS_SECURITY_*`.